genesyscloud/outbound_contact_list_template/resource_genesyscloud_outbound_contact_list_template.go

```diff
@@ -0,0 +1,624 @@
+	})
+}
+
+func buildSdkOutboundContactListTemplateContactPhoneNumberColumnSlice(contactPhoneNumberColumn *schema.Set) *[]platformclientv2.Contactphonenumbercolumn {
+	if contactPhoneNumberColumn == nil {
+		return nil
+	}
+	sdkContactPhoneNumberColumnSlice := make([]platformclientv2.Contactphonenumbercolumn, 0)
+	contactPhoneNumberColumnList := contactPhoneNumberColumn.List()
+	for _, configPhoneColumn := range contactPhoneNumberColumnList {
+		var sdkContactPhoneNumberColumn platformclientv2.Contactphonenumbercolumn
+		contactPhoneNumberColumnMap := configPhoneColumn.(map[string]interface{})
+		if columnName := contactPhoneNumberColumnMap["column_name"].(string); columnName != "" {
+			sdkContactPhoneNumberColumn.ColumnName = &columnName
+		}
+		if varType := contactPhoneNumberColumnMap["type"].(string); varType != "" {
+			sdkContactPhoneNumberColumn.VarType = &varType
+		}
+		if callableTimeColumn := contactPhoneNumberColumnMap["callable_time_column"].(string); callableTimeColumn != "" {
+			sdkContactPhoneNumberColumn.CallableTimeColumn = &callableTimeColumn
+		}
+
+		sdkContactPhoneNumberColumnSlice = append(sdkContactPhoneNumberColumnSlice, sdkContactPhoneNumberColumn)
+	}
+	return &sdkContactPhoneNumberColumnSlice
+}
+
+func flattenSdkOutboundContactListTemplateContactPhoneNumberColumnSlice(contactPhoneNumberColumns []platformclientv2.Contactphonenumbercolumn) *schema.Set {
+	if len(contactPhoneNumberColumns) == 0 {
+		return nil
+	}
+
+	contactPhoneNumberColumnSet := schema.NewSet(schema.HashResource(outboundContactListTemplateContactPhoneNumberColumnResource), []interface{}{})
+	for _, contactPhoneNumberColumn := range contactPhoneNumberColumns {
+		contactPhoneNumberColumnMap := make(map[string]interface{})
+
+		if contactPhoneNumberColumn.ColumnName != nil {
+			contactPhoneNumberColumnMap["column_name"] = *contactPhoneNumberColumn.ColumnName
+		}
+		if contactPhoneNumberColumn.VarType != nil {
+			contactPhoneNumberColumnMap["type"] = *contactPhoneNumberColumn.VarType
+		}
+		if contactPhoneNumberColumn.CallableTimeColumn != nil {
+			contactPhoneNumberColumnMap["callable_time_column"] = *contactPhoneNumberColumn.CallableTimeColumn
+		}
+
+		contactPhoneNumberColumnSet.Add(contactPhoneNumberColumnMap)
+	}
+
+	return contactPhoneNumberColumnSet
+}
+
+func buildSdkOutboundContactListContactEmailAddressColumnSlice(contactEmailAddressColumn *schema.Set) *[]platformclientv2.Emailcolumn {
+	if contactEmailAddressColumn == nil {
+		return nil
+	}
+	sdkContactEmailAddressColumnSlice := make([]platformclientv2.Emailcolumn, 0)
+	contactEmailAddressColumnList := contactEmailAddressColumn.List()
+	for _, configEmailColumn := range contactEmailAddressColumnList {
+		var sdkContactEmailAddressColumn platformclientv2.Emailcolumn
+		contactEmailAddressColumnMap := configEmailColumn.(map[string]interface{})
+		if columnName := contactEmailAddressColumnMap["column_name"].(string); columnName != "" {
```

Comment:
This could throw an interface conversion panic. If `column_name` isn't in the map, `columnName` will be `nil` and casting nil to a string will throw the panic. If you write the if statement like this,

```golang
if columnName, _ := contact ...
```

and the key isn't present, the var will be the zero value (string in this case) for that type.